cmd/mms/main.go

```diff
@@ -75,6 +75,12 @@ func main() {
 			EnvVars: []string{"MMS_EVENT_INTERVAL"},
 			Value:   0,
 		}),
+		altsrc.NewBoolFlag(&cli.BoolFlag{
+			Name:    "insecure",
+			Usage:   "Accept invalid certificates, useful for testing with self-signed ones.",
+			EnvVars: []string{"MMS_INSECURE"},
```

Comment:
Should we really have an environment variable for this? It's better that the user explicitly asks for an insecure connection every time.